acceptor.go

```diff
@@ -98,6 +98,11 @@ func (a *Acceptor) Stop() {
 	a.sessionGroup.Wait()
 }
 
+//GetListenerAddr returns the net.Addr from the listener of the Acceptor
+func (a *Acceptor) GetListenerAddr() net.Addr {
+	return a.listener.Addr()
```

Comment:
unit tests would help, but would be good to handle scenario where GetListenerAddr is called before listener is initialized. Also, just some idiomatic go, lets drop the `Get` from the [getter](https://golang.org/doc/effective_go.html#Getters)